dockers/docker-router-advertiser/docker-router-advertiser.supervisord.conf.j2

```diff
@@ -44,14 +44,16 @@ dependent_startup_wait_for=rsyslogd:running
 {# But not for specific deployment_id #}
 {%- set vlan_v6 = namespace(count=0) -%}
 {%- if DEVICE_METADATA.localhost.deployment_id != "8" -%}
-  {%- if "ToRRouter" in DEVICE_METADATA.localhost.type and DEVICE_METADATA.localhost.type != "MgmtToRRouter" -%}
-    {%- if VLAN_INTERFACE -%}
-      {%- for (name, prefix) in VLAN_INTERFACE|pfx_filter -%}
-        {# If this VLAN has an IPv6 address... #}
-        {%- if prefix | ipv6 -%}
-          {%- set vlan_v6.count = vlan_v6.count + 1 -%}
-        {%- endif -%}
-      {%- endfor -%}
+  {%- if DEVICE_METADATA.localhost.type -%}
```

Comment:
Why would localhost.type not exists. This field is being used in multiple places in Sonic. IMO, this is a missing config. However, changes lgtm. @jleveque for viz.

Reply:
Agree. This change will not hurt anything, but I don't understand a situation where `DEVICE_METADATA.localhost.type` would not exist.

Reply:
Config default_preset field in default_sku to l2, then the generated config_db.json won't have DEVICE_METADATA.localhost.type field.
sonic-cfggen was upgraded to python3. If DEVICE_METADATA.localhost.type is not exist, rendering docker-router-advertiser.supervisord.conf.j2 will fail.